sdk/healthbot/Azure.ResourceManager.HealthBot/tests/TestCase/HealthBotTests.cs

```diff
@@ -0,0 +1,77 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading.Tasks;
+using Azure.Core.TestFramework;
+using Azure.ResourceManager.HealthBot.Models;
+using NUnit.Framework;
+
+namespace Azure.ResourceManager.HealthBot.Tests.TestCase
+{
+    public class HealthBotTests : HealthBotManagementTestBase
+    {
+        public HealthBotTests(bool isAsync)
+            : base(isAsync)//, RecordedTestMode.Record)
+        {
+        }
+
+        public async Task<HealthBotCollection> GetHealthBotCollection()
+        {
+            var group = await CreateResourceGroupAsync();
+            var collection = group.GetHealthBots();
+            return collection;
+        }
+
+        public HealthBotData GetHealthBotData()
+        {
+            var data = new HealthBotData(DefaultLocation, new HealthBotSku(HealthBotSkuName.F0));
+            return data;
+        }
+        public static void AssertData(HealthBotData data1, HealthBotData data2)
+        {
+            Assert.AreEqual(data1.Name, data2.Name);
+            Assert.AreEqual(data1.Id, data2.Id);
+            Assert.AreEqual(data1.Location, data2.Location);
+        }
+
+        [RecordedTest]
+        public async Task HealthBotApiTests()
+        {
+            //1.CreateOrUpdate
+            var collection = await GetHealthBotCollection();
```

Comment:
Please refer the generated samples to add more tests for all resources.

Reply:
fixed